python/frontend/cirrus/cirrus/HyperParamDashDemo.ipynb

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
I think this file shouldn't have been changed. Also the content doesn't look good

```
"name 'app' is not defined",
```

Reply:
This file has been removed (this file DNE in master).